apps/bttester/src/gap.c

```diff
@@ -248,6 +252,40 @@ static struct ble_gap_adv_params adv_params = {
 	.disc_mode = BLE_GAP_DISC_MODE_NON,
 };
 
+#if MYNEWT_VAL(BTTESTER_PRIVACY_MODE) && MYNEWT_VAL(BTTESTER_USE_NRPA)
+static void rotate_nrpa_cb(struct os_event *ev)
+{
+    assert(ev != NULL);
```

Comment:
this assert seems to be not needed. Otherwise looks good. Please remove it and feel free to merge